Cannot install dependencies using npm@7.

As discussed in #24, this package's dependencies can't be installed using npm@7 unless you use the `--legacy-peer-deps` flag, which is brought in by the `stylelint-config-fluid` dependency.    That package is using the latest version of all its dependencies, but `stylelint-config-recommended` and others haven't been updated to work with newer versions of `stylelint`.

```
npm ERR! 
npm ERR! Found: stylelint@13.12.0
npm ERR! node_modules/stylelint-config-fluid/node_modules/stylelint
npm ERR!   stylelint@"13.12.0" from stylelint-config-fluid@0.2.0
npm ERR!   node_modules/stylelint-config-fluid
npm ERR!     stylelint-config-fluid@"0.2.0" from the root project
npm ERR!   peer stylelint@"^13.12.0" from stylelint-config-standard@21.0.0
npm ERR!   node_modules/stylelint-config-fluid/node_modules/stylelint-config-standard
npm ERR!     stylelint-config-standard@"21.0.0" from stylelint-config-fluid@0.2.0
npm ERR!     node_modules/stylelint-config-fluid
npm ERR!       stylelint-config-fluid@"0.2.0" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer stylelint@"^8.3.0 || ^9.0.0 || ^10.0.0" from stylelint-config-recommended@2.2.0
npm ERR! node_modules/stylelint-config-fluid/node_modules/stylelint-config-standard-scss/node_modules/stylelint-config-recommended
npm ERR!   stylelint-config-recommended@"^2.2.0" from stylelint-config-standard@18.3.0
npm ERR!   node_modules/stylelint-config-fluid/node_modules/stylelint-config-standard-scss/node_modules/stylelint-config-standard
npm ERR!     stylelint-config-standard@"~18.3.0" from stylelint-config-standard-scss@1.1.0
npm ERR!     node_modules/stylelint-config-fluid/node_modules/stylelint-config-standard-scss
npm ERR!       stylelint-config-standard-scss@"1.1.0" from stylelint-config-fluid@0.2.0
```

--- a/.github/workflows/main.yml
+++ b/.github/workflows/main.yml
@@ -26,9 +26,6 @@ jobs:
         with:
           node-version: ${{ matrix.node-version }}
 
-      - name: Use NPM 6.x
-        run: npm install -g npm@6
-
       - name: Install Node.js dependencies
         run: npm install
 

--- a/package.json
+++ b/package.json
@@ -36,8 +36,8 @@
         "markdownlint-config-fluid": "0.1.3",
         "minimatch": "3.0.4",
         "minimist": "1.2.5",
-        "stylelint": "13.9.0",
-        "stylelint-config-fluid": "0.1.0"
+        "stylelint": "13.13.1",
+        "stylelint-config-fluid": "0.2.1"
     },
     "devDependencies": {
         "node-jqunit": "1.1.8"

